api/src/db/seedingDocs/lang-eng.json

```
@@ -80,6 +80,7 @@
         "logout.modal.title": "Logout",
         "logout.modal.description": "Are you sure you want to log out?",
         "logout.modal.button_cancel": "Cancel",
-        "logout.modal.button_logout": "Logout"
+        "logout.modal.button_logout": "Logout",
+        "navigation.cta.home": "Back to home"
```

```suggestion
        "notfoundpage.navigation.home": "Back to home"
```
We have to think about the translators, that they will easily be able to know where the text is use din the app